### DISCUSSION TITLE
Edit this page on Github link

### DISCUSSION THREAD

**Discussion by arky** [thuliteio/doks#1248] [Q&A] [answered]
Could someone point me to right page, partial that renders the `Edit this page on Github` links in content pages.  

Am trying to fix bug in following [repo](https://github.com/PiRogueToolSuite/piroguetoolsuite.github.io) with following config. 

```
  # Repository
  editPage = true # false (default) or true
  lastMod = true # false (default) or true
  repoHost = "GitHub" # GitHub (default), Gitea, GitLab, Bitbucket, or BitbucketServer
  docsRepo = "https://github.com/PiRogueToolSuite/piroguetoolsuite.github.io"
  docsRepoBranch = "v2" # main (default), master, or <branch name>
  docsRepoSubPath = "" # "" (none, default) or <sub path>

```

**Comment by h-enk** [comment, marked as answer]
That's `node_modules/layouts/partials/main/last-modified.html`. Copy and paste as `layouts/partials/main/last-modified.html` and make your modifications.

See also:

- https://github.com/orgs/gethyas/discussions/454
- https://github.com/gethyas/doks-core/issues/57

**Reply by arky** [reply]
Thank you @h-enk

**Reply by h-enk** [reply]
You're welcome!
